src/parser/generated_parser.ts

```diff
@@ -2497,7 +2497,7 @@ export class GeneratedParser extends Parser {
         }
         this.reset(mark);
         if ((keyword = this.expect("__peg_parser__"))) {
-            return RAISE_SYNTAX_ERROR("You found it!");
+            return pegen.raise_error(this, pySyntaxError, "You found it!");
```

Comment:
maybe `raise_error` and `raise_error_at_known_location` should really be on the `Parser`
so that we can call
`this.raise_error` without having to pass this as the first argument.

Reply:
this is true for the whole of pegen.ts

Reply:
yes - but I think we actually need `this` here where as I think for all the functions we've written so far we haven't used the `p` argument.

Reply:
https://twitter.com/ajpnijburg/status/1408732658037923843?s=20

Reply:
Oh very interesting - great use of Twitter. What does that mean for us do you think?

(I think it means we've got it right with `name`, `expect` etc)

(And we still need to work out `fill_token` does for us).

Reply:
from several expeditions down the `fill_token` rabbit hole, I think the most important thing for us is finding `# type: ignore` comments